test/performance/specs/front-end-block-theme.spec.js

```diff
@@ -19,19 +14,27 @@ test.describe( 'Front End Performance', () => {
 		await requestUtils.activateTheme( 'twentytwentythree' );
 	} );
 
-	test.afterAll( async ( { requestUtils } ) => {
-		saveResultsFile( __filename, results );
+	test.afterAll( async ( { requestUtils }, testInfo ) => {
+		await testInfo.attach( 'results', {
+			body: JSON.stringify( results, null, 2 ),
+			contentType: 'application/json',
+		} );
 		await requestUtils.activateTheme( 'twentytwentyone' );
 	} );
 
-	test( 'Report TTFB, LCP, and LCP-TTFB', async ( { page } ) => {
-		let i = 16;
-		while ( i-- ) {
+	test( 'Report TTFB, LCP, and LCP-TTFB', async ( { browser } ) => {
+		const samples = 16;
+		const throwaway = 0;
+		const rounds = samples + throwaway;
+
+		for ( let i = 0; i < rounds; i++ ) {
+			const testPage = await browser.newPage( { storageState: {} } );
```

Comment:
It's [recommended](https://playwright.dev/docs/api/class-browser#browser-new-page) to spawn a new `context` than a new page directly so that the lifecycle of the context can be better controlled.